### ISSUE TITLE
Optifabric crashing game 

### ISSUE THREAD

**Issue by 0DarkShock0** [Chocohead/OptiFabric#782]
Optifabric is crashing minecraft with exit code 1 as soon as the game is launched. 
I am using the following version: (optifabric- 1.13.5) and i'm using fabric loader-0.14.8-1.18.2 to launch the game
Also, the optifine version is 1.18.2_HD_U_H7

**Comment by TheReaperT04** [comment]
Yeah optifabric doesn't seem to be working with newest version of optifine atm

**Comment by Chocohead** [comment]
Going to need a mod list you're using, or ideally the log file/crash report.

**Comment by HarryRich02** [comment]
I am also having this issue. Attached is the log.
[latest.log](https://github.com/Chocohead/OptiFabric/files/9190136/latest.log)
.

**Comment by topia** [comment]
> I am also having this issue. Attached is the log.
> [latest.log](https://github.com/Chocohead/OptiFabric/files/9190136/latest.log)
> .

https://github.com/Chocohead/OptiFabric/issues/800#issuecomment-1193835896

**Comment by HarryRich02** [comment]
Yes, this seems to be an incompatibility with inventory sorter then.
